### ISSUE TITLE
Velociraptor DDNS inoperable

### ISSUE THREAD

**Issue by dd8917vk** [Velocidex/velociraptor#1926]
Velociraptor DDNS (with google DDNS) does not work in its current state.  Velociraptor uses myexternalip.com to resolve IP addresses.  myexternalip.com currently shows everyone's external IP to be 

<img width="734" alt="image" src="https://user-images.githubusercontent.com/59884681/177647113-ed6cac3f-0e40-44d7-87d6-a0b99b334023.png">

This breaks all servers deployed with google DDNS.  Mike Cohen is aware of the issue.

**Comment by mgreen27** [comment]
Thank you for reporting this! 
The current workaround is to simply turn off DDNS and manually set the public IP on your domain.

The velociraptor config file has a ddns configuration if configured:

```
  dyn_dns:
    ddns_username: <USERNAME>
    ddns_password: <PASSWORD>
```

Remove this configuration and restart the service.
Add in correct IP to your Google DNS config.

**Comment by mgreen27** [comment]
Just an update: looks like myexternalip.com is resolving correctly now.

**Comment by scudette** [comment]
This should be fixed by #1933 which changed the default external ip check service to google domains as well as make it configurable. From what i understand on this ticket this was just a temporary error with myexternalip.com which is now resolved? if so there is no urgency to backport this fix to 0.6.5 and we can wait for 0.6.6 to include this change
